.github/workflows/release.yml

```diff
@@ -0,0 +1,127 @@
+on:
+  workflow_dispatch:
+    inputs:
+      ReleaseType:
+        type: choice
+        description: Select the version to released
+        options:
+          - Major Version
+          - Minor Version
+          - Patch Version
+
+env:
+  BOT_USER_NAME: eclipse-xpanse-bot
+  BOT_EMAIL_ID: xpanse-bot@eclipse.org
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  pre-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set current version env variable
+        id: version
+        run: |
+          echo "CurrentVersion=$(sed -n 's/^.*version.*= *"\([a-z0-9.]*\)"/\1/pg' version.go)" >> $GITHUB_OUTPUT
+          if [ "${{github.event.inputs.ReleaseType}}" = "Major Version" ]; then
+            echo "VersionFragment=major" >> $GITHUB_ENV
+          elif [ "${{github.event.inputs.ReleaseType}}" = "Minor Version" ]; then
+            echo "VersionFragment=feature" >> $GITHUB_ENV
+          elif [ "${{github.event.inputs.ReleaseType}}" = "Patch Version" ]; then
+            echo "VersionFragment=bug" >> $GITHUB_ENV
+          else
+            echo "No matching feature type  found"
+          fi
+      - name: Set next development version environment variable
+        id: new_version
+        uses: christian-draeger/increment-semantic-version@1.1.0
+        with:
+          current-version: ${{ steps.version.outputs.CurrentVersion }}
+          version-fragment: ${{ env.VersionFragment }}
+      - name: Update version
+        run: |
+          sed -i 's/\(^.*version.*= *"\)\([a-z0-9.]*\)"/\1${{ steps.new_version.outputs.next-version }}"/g' version.go
+      - uses: EndBug/add-and-commit@v9
+        with:
+          message: Release v${{ steps.new_version.outputs.next-version }}
+          committer_name: ${{ env.BOT_USER_NAME }}
+          committer_email: ${{ env.BOT_EMAIL_ID }}
+      - name: Create a GitHub release
+        uses: ncipollo/release-action@v1
+        with:
+          tag: v${{ steps.new_version.outputs.next-version }}
+          name: Release v${{ steps.new_version.outputs.next-version }}
+    outputs:
+      current_version: ${{ steps.version.outputs.CurrentVersion }}
+      release_version: v${{ steps.new_version.outputs.next-version }}
+  docker-releaser:
+    needs: pre-release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{ needs.pre-release.outputs.release_version }}
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ">=1.21"
+      - name: Build policy-man
+        run: make build
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to Github Packages
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ env.BOT_USER_NAME }}
+          password: ${{ secrets.GHCR_TOKEN }}
```

Comment:
we must use "secrets.BOT_GITHUB_DOCKER_TOKEN" token here. This token is available for all repos in eclipse-xpanse org.

Reply:
> we must use "secrets.BOT_GITHUB_DOCKER_TOKEN" token here. This token is available for all repos in eclipse-xpanse org.

yes, it should be